src/Utilities/for_testing/CMakeLists.txt

```diff
@@ -0,0 +1,21 @@
+#//////////////////////////////////////////////////////////////////////////////////////
+#// This file is distributed under the University of Illinois/NCSA Open Source License.
+#// See LICENSE file in top directory for details.
+#//
+#// Copyright (c) 2021 QMCPACK developers.
+#//
+#// File developed by: Peter Doak, , doakpw@ornl.gov, Oak Ridge National Laboratory
+#//////////////////////////////////////////////////////////////////////////////////////
+
+SET(test_utilities_src
+  RandomForTest.cpp)
+
+if(USE_OBJECT_TARGET)
```

Comment:
This is not needed. USE_OBJECT_TARGET is only needed if it depends on a target with OpenMP offload inside.